file_store/src/lora_beacon_report.rs

```diff
@@ -34,6 +35,26 @@ impl MsgDecode for LoraBeaconIngestReport {
     type Msg = LoraBeaconIngestReportV1;
 }
 
+impl LoraBeaconIngestReport {
```

Comment:
this looks like a great opportunity for an `ReportId` trait? maybe a TODO if it's not worth adding right now?

Reply:
yeah was originally going to add as a trait but need to get it in...will keep it as an improvement